index.js

```diff
@@ -10,6 +13,12 @@ module.exports = function postcssPrefixSelector(options) {
     ) {
       return;
     }
+    if (
+      root.source.input.file &&
+      includeFiles.some((file) => !root.source.input.file.includes(file))
```

Comment:
This won't work as expected, if the `includeFiles` array has two entries for example 'a.css' and `b.css` this will return `false`.